scripts/managedb/csubs_control/README.md

```diff
@@ -0,0 +1,20 @@
+This directory includes some scripts to help to control context subscriptions collection (csubs). All them share
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/7115621dcc90ba36dee2235ad4ccc909ee1b11d3